crates/bevy_render/src/camera/camera.rs

```diff
@@ -221,127 +244,56 @@ pub fn camera_system<T: CameraProjection + Component>(
             || added_cameras.contains(&entity)
             || camera_projection.is_changed()
         {
-            if let Some(size) = camera.target.get_logical_size(&windows, &images) {
+            camera.logical_target_size = camera.target.get_logical_size(&windows, &images);
+            camera.physical_target_size = camera.target.get_physical_size(&windows, &images);
+            if let Some(size) = camera.logical_target_size {
                 camera_projection.update(size.x, size.y);
+                camera.logical_target_size = Some(size);
```

Comment:
Why? This is assigning the value it already has?

Reply:
Nice catch. Missed this during a refactor.